scripts/setup_host.sh

```diff
@@ -543,6 +551,7 @@ ubu_update_fw
 install_vm_manager
 
 prepare_required_scripts
+start_thermal_daemon
```

Comment:
is it really always needed? I think this is an optional case

Reply:
thermal related epics will not work, if this change is not available.